test/65285_temperature.js

```diff
@@ -0,0 +1,62 @@
+var chai = require('chai')
+chai.Should()
+chai.use(require('chai-things'))
+chai.use(require('@signalk/signalk-schema').chaiModule)
+var debug = require('debug')('n2k-signalk:test:130312')
+var _ = require('lodash')
+
+var msgs = require('./65285_temperature-2.json')
+
+describe('Temperature: ', function () {
+  it('examples work', function () {
+    var n2kMapper = require('../n2kMapper.js')
+    var full = new (require('@signalk/signalk-schema')).FullSignalK(
+      'urn:mrn:imo:mmsi:230099999'
+    )
+    _.forOwn(msgs, function (msg, key) {
+      var delta = n2kMapper.toDelta(msg)
+      delta.context = 'vessels.urn:mrn:imo:mmsi:230099999'
+      full.addDelta(delta)
+      delta.should.be.validSignalKDelta
+      delta.updates[0].values.forEach(function (pathValue) {})
```

Comment:
This does not do anything?

Reply:
Hmm I just tried to mock `13031_temperature.js`. Going to look at it again.

Reply:
Yes, that test is shaped like that to test that the different 13031x samples in https://github.com/SignalK/n2k-signalk/blob/master/test/130310-2.json all result in valid SK output. Probably should add some test there that checks for correct output values as well.